package.json

```diff
@@ -7,12 +7,15 @@
   "scripts": {
     "backend:dev": "yarn workspace backend run dev",
     "web:dev": "yarn workspace web run dev",
+    "migrate": "yarn workspace backend migrate",
+    "worker": "yarn workspace backend worker",
     "mobile:dev": "yarn workspace mobile start",
     "mobile:pod-install": "yarn workspace mobile pod-install",
     "mobile:android": "yarn workspace mobile run android",
     "mobile:clean": "yarn workspace mobile run clean",
     "mobile:ios": "yarn workspace mobile run ios",
-    "reset": "rm -rf **/node_modules **/.yarn/cache **/yarn.lock && yarn install"
+    "reset": "rm -rf **/node_modules **/.yarn/cache **/yarn.lock && yarn install",
+    "heroku-postbuild": "yarn workspace web build && yarn workspace @hylo/shared build:cjs"
```

Comment:
So this hook runs after the build on Heorku for both backend and web? If this works in deploy can we leave the on-the-fly @hylo/shared build for the `yarn workspace backend dev`?

Reply:
ok. let's do that and then will revisit the deploy path in a bit. so currently the post build hook does the job for the deploy?

Reply:
ideally the postbuild uses a script that detects from an environment variable whether we are doing a frontend or back-end deploy and only builds the web for frontend deploys. but builds shared for both. that is a later task